js/src/lib/interfaces/Libraries.ts

```diff
@@ -233,14 +233,24 @@ model = timm.create_model("hf_hub:${model.id}", pretrained=True)`;
 const sklearn = (model: ModelData) => {
 	if (model.tags?.includes("skops")) {
 		const skopsmodelFile = model.config?.sklearn?.filename;
+		const skopssaveFormat = model.config?.sklearn?.model_format;
+		if (skopssaveFormat == "pickle")
+		{
+			return `import joblib
+from skops.hub_utils import download
+
+download("${model.id}", "path_to_folder")
+model = joblib.load(
+	"${skopsmodelFile}"
+)`
+} else {
 		return `from skops.hub_utils import download
 from skops.io import load
 
 download("${model.id}", "path_to_folder")
-# make sure model file is in skops format
-# if model is a pickle file, make sure it's from a source you trust
 model = load("path_to_folder/${skopsmodelFile}")`;
-	} else {
+	}
+ } else {
```

Comment:
Please unindent the closing `)` in the vanilla sklearn code snippet

Reply:
Done